lua/LuaLibScript.h

```diff
@@ -34,7 +34,7 @@ class LuaLibScript
         bool call_on_internal_chain_event_callbacks(InternalEvents::Identifiers, int nArg=0, int nRet=0);
 
         int call_on_render_event_pre_callbacks(RenderEvents::Identifiers, int nArg);
-        void call_on_render_event_post_callbacks(RenderEvents::Identifiers, unsigned int idx, int nArg);
```

Comment:
Why did you remove the `unsigned`? This index value is never negative, it's an array index, it must always be `>= 0`.

Reply:
Or I guess, describe how this causes an infinite loop, because I don't understand what the heck you mean.

Reply:
The loop continues as long as idx >= 0, which is always true for an unsigned int.

Reply:
I'll need to look at my code, but this doesn't seem like the ideal way to fix it if it's a logic bug, the API should remain that it's an index number of an array which is bounded by 0.

Reply:
I was kind of afraid to touch the logic, I didn't completely get it. Shall I undo this change for this PR?

Reply:
I undid it